Procfile

```diff
@@ -1 +0,0 @@
-web: python -m girder --port ${PORT}
```

Comment:
There is a section in our deployment docs about Heroku, and it mentions these files and kind of makes it sound like they are important to deploy onto Heroku. @cpatrick might know whether these are still required.